frontend/src/SignUp.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import './SignUp.css';
+
+
+function SignUp() {
+  return (    
+    <div >
+
```

Comment:
let's run the eslint in the `frontend` directory so we can get correct nested formatting within this file!

see: https://stackoverflow.com/a/66438578

Reply:
So it didn't do much after running "npm run lint" in the front-end directory. I ran "npm i" in case and then ran 'npm run lint' again. That changed nothing? I also added it to the package.json under scripts but I doubt that was necessary.  What should be happening when I run 'npm run lint'?

Reply:
actually you're right, i was thinking of `prettier` not eslint. you can ignore this comment but if you're up for it, consider adding prettier in a followup PR

https://medium.com/@frontend-newbie/integrating-prettier-with-eslint-for-your-create-react-app-in-vscode-153ebe89c1a2